.gitignore

```diff
@@ -1,5 +1,6 @@
 # Shared test files
-src/test/resources/
+src/test/resources/assignment-v2
+src/test/resources/rac-experiments-v3.json
```

Comment:
Specifically ignoring the common test files so that I can check in additional, bandit-related test files.